testing/test_collection.py

```diff
@@ -628,9 +630,9 @@ def test_method(self):
 
 class Test_getinitialnodes:
     def test_global_file(self, pytester: Pytester) -> None:
-        tmpdir = pytester.path
-        x = ensure_file(tmpdir / "x.py")
-        with tmpdir.cwd():
+        tmp_path = pytester.path
+        x = ensure_file(tmp_path / "x.py")
+        with tmp_path.cwd():
```

Comment:
This doesn't do the same thing. In `py.path`, the `cwd()` context manager does a `chdir()` to the path on enter and restores to previous value on exit. In `pathlib`, the context manager doesn't do anything (I have no idea why it even exists...).

The replacement is something like

```py
with MonkeyPatch.context() as mp:
    mp.chdir(tmp_path)
```

Although now that I think of it, `pytester` already does a `chdir` into its path, so the `with` is actually unnecessary and you can just remove it :smile: